modules/crud/sql/transaction.go

```diff
@@ -0,0 +1,65 @@
+package sql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/spaceuptech/space-cloud/model"
+)
+
+func (s *SQL) Transaction(ctx context.Context, project string, reqs *model.TransactionRequest) error {
+
+	var opts *sql.TxOptions
+	tx, err := s.client.BeginTxx(ctx, opts)
```

Comment:
There are no tx rollbacks happening in case of errors. Lets do that.

Reply:
Sqlx automatically handels rollback function 
 " If the context is canceled, the sql package will roll back the transaction automatically " --> GODOC

Reply:
You are right!